config/crd/bases/certmanager.cloud.sap_digicertissuers.yaml

```diff
@@ -93,13 +97,21 @@ spec:
                   skipApproval:
                     description: SkipApproval skips the approval of the certificate.
                     type: boolean
+                  validityDays:
+                    description: ValidityDays is the validity of the order and certificate
+                      in days. Overrides ValidityYears if set.
+                    type: integer
                   validityYears:
-                    description: ValidityYears is the validity of the certificate
-                      in years.
+                    description: |-
+                      ValidityYears is the validity of the order and certificate in years. Defaults to 1 year if not set.
+                      Can be overridden by ValidityDays.
                     type: integer
                 required:
                 - apiTokenReference
                 type: object
+                x-kubernetes-validations:
+                - message: only one of validityDays and validityYears can be set.
+                  rule: '!has(self.validityDays) || !has(self.validityYears)'
```

Comment:
```suggestion
                  rule: '(has(self.validityDays) && !has(self.validityYears)) || (!has(self.validityDays) && has(self.validityYears))'
```

Reply:
I adapted the validation, hope it is more readable now.